apps/root/src/pages/token-profile/frame/index.tsx

```diff
@@ -36,11 +39,12 @@ const TokenProfileFrame = () => {
   });
 
   React.useEffect(() => {
+    dispatch(changeRoute('token'));
```

Comment:
can we use the ROUTE key for this?

Reply:
Sure, also this can be improved in 'history' page